.drone.yml

```diff
@@ -0,0 +1,198 @@
+    detach: true
+    pull: true
+    commands:
+      - tail -f /var/www/owncloud/data/owncloud.log
+
+  code-compliance-check:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/apps/notifications
+      - make test-php-codecheck
+      - make test-php-lint
+
+  fix-permissions:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/
+      - chown www-data * -R
+
+  webui-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - BROWSER=chrome #chrome or firefox
+      - SELENIUM_HOST=selenium
+      - SRV_HOST_NAME=owncloud
+      - SRV_HOST_PORT=80
+      - SKELETON_DIR=/var/www/owncloud/tests/acceptance/webUISkeleton
+      - SELENIUM_PORT=4444
+      - PLATFORM=Linux
+      - APPS_TO_ENABLE=notifications
+    commands:
+      - cd /var/www/owncloud
+      - chmod 777 /var/www/owncloud/tests/acceptance/filesForUpload -R
+      - bash tests/travis/start_ui_tests.sh --remote --config /var/www/owncloud/apps/notifications/tests/acceptance/config/behat.yml
+    when:
+      matrix:
+        TEST_SUITE: web-acceptance
+
+  api-acceptance-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/apps/notifications/tests/acceptance
+      - bash ./run.sh apiNotifications
+    when:
+      matrix:
+        TEST_SUITE: api-acceptance
+
+  phpunit-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - PHP_VERSION=${PHP_VERSION}
+    commands:
+      - cd /var/www/owncloud
+      - phpdbg -d memory_limit=4096M -rr ./lib/composer/bin/phpunit --configuration apps/notifications/tests/Unit/phpunit.xml
+      - cd apps/notifications/
+      - wget -q https://scrutinizer-ci.com/ocular.phar
+      - php ocular.phar code-coverage:upload --format=php-clover tests/Unit/clover.xml
+    when:
+      matrix:
+        TEST_SUITE: phpunit
+
+  codecov:
+    image: robertstettner/drone-codecov
+    pull: true
+    files:
+      - apps/notifications/Unit/clover.xml
+    token: cb3d43ae-a52a-4d88-83d5-04a701de1007
+    when:
+      matrix:
+        TEST_SUITE: phpunit
+        PHP_VERSION: 7.1
+
+services:
+  owncloud:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    environment:
+      - APACHE_WEBROOT=/var/www/owncloud/
+    command: [ "/usr/local/bin/apachectl", "-e", "debug", "-D", "FOREGROUND" ]
+    when:
+      matrix:
+        TEST_SUITE: web-acceptance
+
+  selenium:
+    image: selenium/standalone-chrome-debug:latest
+    pull: true
+    when:
+      matrix:
+        TEST_SUITE: web-acceptance
+
+matrix:
+  include:
+    - PHP_VERSION: 7.1
+      OC_VERSION: master
+      TEST_SUITE: phpunit
+      DB_TYPE: mysql
+    - PHP_VERSION: 7.1
+      OC_VERSION: master
+      TEST_SUITE: phpunit
+      DB_TYPE: pgsql
+    - PHP_VERSION: 7.1
+      OC_VERSION: master
+      TEST_SUITE: phpunit
+      DB_TYPE: oracle
```

Comment:
`oracle` is not the driver owncloud knows about - ref: https://github.com/owncloud/core/blob/92b658a60e6c28ef9dd7594384180e0d73e96660/lib/private/Setup.php#L85-L89